datafusion/sql/src/expr/function.rs

```diff
@@ -93,6 +93,8 @@ struct FunctionArgs {
     distinct: bool,
     /// WITHIN GROUP clause, if any
     within_group: Vec<OrderByExpr>,
+    /// Was the function called without parenthesis, i.e. could this also be a column reference?
+    function_without_paranthesis: bool,
```

Comment:
I think there is a typo here:

```suggestion
    function_without_parenthesis: bool,
```